### PR TITLE
Add option to read GB wind from other report

### PR DESCRIPTION
**Pull request by lorrieq**
Addresses issue #1754 

Controlled by constant `FETCH_WIND_FROM_FUELINST`.

The GB wind in report B1620 is still acting up, not just way too low now but jumping all over the place.
em_gb_wind

### CODE REVIEW

**Comment by brunolajoie** [comment]
The following error is generated when testing your branch locally
image

File: parsers/ELEXON.py [hunk 1]
```diff
@@ -218,6 +224,48 @@ def datetime_from_date_sp(date, sp):
...
+    end = target_datetime + dt.timedelta(hours=12)
+
+    session = requests.session()
+    params = {
+        'FromDateTime': start.strftime('%Y-%m-%d %H:%M:%S'),
+        'ToDateTime': end.strftime('%Y-%m-%d %H:%M:%S'),
+        'ServiceType': 'csv'
+    }
+    response = query_ELEXON('FUELINST', session, params)
+    csv_text = response.content
```
Comment (systemcatch) [inline parsers/ELEXON.py]: I think if this is changed to `csv_text = response.text` it should fix the error.

**Comment by lorrieq** [comment]
Apologies @brunolajoie, this change should fix it as @systemcatch said! I was too lazy to setup Python 3 environment on other computer and tested it off Python 2. Anyway I will set that up for future.

**Comment by brunolajoie** [comment]
Nice! It seems to work when I test it, however, i got some "None" answer from Wind which looks supsicious

`{'zoneKey': 'GB', 'datetime': datetime.datetime(2019, 2, 18, 2, 30, tzinfo=tzfile('/usr/share/zoneinfo/Europe/London')), 'source': 'bmreports.com', 'production': {'biomass': 1084.0, 'hydro': 400.0, 'coal': 0.0, 'gas': 4868.0, 'oil': 0.0, 'nuclear': 5970.0, 'unknown': 69.0, 'wind': 8948, 'solar': 0.0}, 'storage': {'hydro': -0.0}}, {'zoneKey': 'GB', 'datetime': datetime.datetime(2019, 2, 18, 2, 0, tzinfo=tzfile('/usr/share/zoneinfo/Europe/London')), 'source': 'bmreports.com', 'production': {'biomass': 1090.0, 'hydro': 386.0, 'coal': 0.0, 'gas': 4620.0, 'oil': 0.0, 'nuclear': 5961.0, 'unknown': 71.0, 'wind': 9044, 'solar': 0.0}, 'storage': {'hydro': -0.0}}, {'zoneKey': 'GB', 'datetime': datetime.datetime(2019, 2, 18, 1, 30, tzinfo=tzfile('/usr/share/zoneinfo/Europe/London')), 'source': 'bmreports.com', 'production': {'biomass': 1068.0, 'hydro': 416.0, 'coal': 0.0, 'gas': 4519.0, 'oil': 0.0, 'nuclear': 5967.0, 'unknown': 71.0, 'wind': 9229, 'solar': 0.0}, 'storage': {'hydro': -0.0}}, {'zoneKey': 'GB', 'datetime': datetime.datetime(2019, 2, 18, 1, 0, tzinfo=tzfile('/usr/share/zoneinfo/Europe/London')), 'source': 'bmreports.com', 'production': {'biomass': 1151.0, 'hydro': 415.0, 'coal': 0.0, 'gas': 4596.0, 'oil': 0.0, 'nuclear': 5964.0, 'unknown': 71.0, 'wind': None, 'solar': 0.0}, 'storage': {'hydro': -0.0}}, {'zoneKey': 'GB', 'datetime': datetime.datetime(2019, 2, 18, 0, 30, tzinfo=tzfile('/usr/share/zoneinfo/Europe/London')), 'source': 'bmreports.com', 'production': {'biomass': 1148.0, 'hydro': 411.0, 'coal': 0.0, 'gas': 4118.0, 'oil': 0.0, 'nuclear': 5966.0, 'unknown': 71.0, 'wind': None, 'solar': 0.0}, 'storage': {'hydro': -0.0}}, {'zoneKey': 'GB', 'datetime': datetime.datetime(2019, 2, 18, 0, 0, tzinfo=tzfile('/usr/share/zoneinfo/Europe/London')), 'source': 'bmreports.com', 'production': {'biomass': 1163.0, 'hydro': 402.0, 'coal': 0.0, 'gas': 3988.0, 'oil': 0.0, 'nuclear': 5960.0, 'unknown': 71.0, 'wind': None, 'solar': 0.0}, 'storage': {'hydro': -0.0}`

**Comment by lorrieq** [comment]
Ah ok, that happens when I join the wind report back on the other fuel report. If it can't find a time match in the wind report then it sets wind to None.

In this case it's happening because the wind part only fetches the last twelve hours of data and the other report fetches the current day.
I will change the wind part to fetch the same time range as the primary report.

**Comment by brunolajoie** [comment]
This one looks good

{'zoneKey': 'GB', 'datetime': datetime.datetime(2019, 2, 18, 0, 30, tzinfo=tzfile('/usr/share/zoneinfo/Europe/London')), 'source': 'bmreports.com', 'production': {'biomass': 1148.0, 'hydro': 411.0, 'coal': 0.0, 'gas': 4118.0, 'oil': 0.0, 'nuclear': 5966.0, 'unknown': 71.0, 'wind': 9534, 'solar': 0.0}, 'storage': {'hydro': -0.0}}, {'zoneKey': 'GB', 'datetime': datetime.datetime(2019, 2, 18, 0, 0, tzinfo=tzfile('/usr/share/zoneinfo/Europe/London')), 'source': 'bmreports.com', 'production': {'biomass': 1163.0, 'hydro': 402.0, 'coal': 0.0, 'gas': 3988.0, 'oil': 0.0, 'nuclear': 5960.0, 'unknown': 71.0, 'wind': 9630, 'solar': 0.0}, 'storage': {'hydro': -0.0}}]
---------------------
took 1.76s
min returned datetime: 2019-02-18T00:00:00+00:00 UTC
max returned datetime: 2019-02-18T14:30:00+00:00 UTC  -- OK, <2h from now :) (now=2019-02-18T16:18:49.479417+00:00 UTC)

**Pull request merged**